content/posts/newsletter/2020-02-04.md

```diff
@@ -0,0 +1,262 @@
+* XEP-0409 (IM Routing-NG)
+* XEP-0411 (Bookmarks Conversion)
+
+### Proposed 
+
+Title: Inbox
+Abstract: This specification proposes a mechanism by which clients can find a list of ongoing conversations and their state.
+URL: https://xmpp.org/extensions/inbox/inbox.html
+
+Title: Full Text Search in MAM
+Abstract: This specification proposes a field in the MAM form for full text searching.
+URL: https://xmpp.org/extensions/inbox/fulltext.html
+
+### New 
+
+Version 0.1.0 of XEP-0427 (MAM Fastening Collation) has been released.
+
+Abstract:
+This specification proposes a mechanism by which MAM results
+containing fastenings can be collated effectively.
+
+URL: https://xmpp.org/extensions/xep-0427.html
+
+Version 0.1.0 of XEP-0428 (Fallback Indication) has been released.
+
+Abstract:
+This specification proposes a mechanism by which message bodies can be
+marked as being purely for fallback purposes, and therefore to be
+ignored by intermediaries and anything that understands the remainder
+of the message.
+
+URL: https://xmpp.org/extensions/xep-0428.html
+
+Version 0.1.0 of XEP-0429 (Special Interests Group End to End Encryption) has been released.
+
+Abstract:
+This document proposes the formation of a Special Interest Group (SIG)
+within the XSF devoted to the development of end-to-end encryption
+within the context of XMPP.
+
+URL: https://xmpp.org/extensions/xep-0429.html
+
+Version 0.1.0 of XEP-0431 (Full Text Search in MAM) has been released.
+
+Abstract:
+This specification proposes a field in the MAM form for full text
+searching.
+
+URL: https://xmpp.org/extensions/xep-0431.html
+
+Version 0.1.0 of XEP-0430 (Inbox) has been released.
+
+Abstract:
+This specification proposes a mechanism by which clients can find a
+list of ongoing conversations and their state.
+
+URL: https://xmpp.org/extensions/xep-0430.html
+
+### Updated 
+
+* Version 0.3.0 of XEP-0401 (Easy User Onboarding)
+* Version 0.2.0 of XEP-0426 (Character counting in message bodies)
+* Version 0.2.1 of XEP-0393 (Message Styling)
+* Version 0.10.0 of XEP-0363 (HTTP File Upload)
+* Version 0.2.1 of XEP-0425 (Message Moderation)
+* Version 0.2.0 of XEP-0429 (Special Interests Group End to End Encryption)
+* Version 0.2.0 of XEP-0431 (Full Text Search in MAM)
+
+### Obsoleted 
+
+Version 1.0.1 of XEP-0412 (XMPP Compliance Suites 2019)
+URL: https://xmpp.org/extensions/xep-0412.html
+
+### Last Call 
+
+This message constitutes notice of a Last Call for comments on XEP-0363. The 
+Last Call was restarted after the Council election, because the previous 
+Council did not vote on the ongoing LC.
+
+Title: HTTP File Upload
+
+Abstract:
+This specification defines a protocol to request permissions from
+another entity to upload a file to a specific path on an HTTP server
+and at the same time receive a URL from which that file can later be
+downloaded again.
+
+URL: https://xmpp.org/extensions/xep-0363.html
+
+This Last Call begins today and shall end at the close of business on
```

Comment:
"begins today" doesn't make sense in the newsletter. Better write a date range ("Last Call went from 2020-01-14 to 2020-01-28")